### DISCUSSION TITLE
New release

### DISCUSSION THREAD

**Discussion by abcdw** [magit/magit#4742] [General]
It's been a while since 3.3.0 and a few fixes and improvements appeared since then, for example https://github.com/alphapapa/magit-todos/issues/114 now resolved and probably many others. It would be cool to have a new tag, so magit package could be updated in distribution repositories. I would be glad to update package definition in GNU Guix once tag is placed.

**Comment by tarsius** [comment]
See my status. Doing this is one of the first things I plan to do when I get back.

**Reply by abcdw** [reply]
Oh, I see.  Wish you a high-quality recovery and great health!

**Comment by yasuhirokimura** [comment]
Would you mind my asking you about current status of new release?

**Reply by tarsius** [reply]
The plan was in 23 minutes, i.e., 1th of July, but I won't make it.
Probably gonna take me another 2-3 weeks.

**Reply by tarsius** [reply]
(Yesterday I did manage to land the last feature that was planned for before the release and am now where I wanted to be ~ two weeks ago. Now I'll have to fix some regression, let some more time pass to let other regressions bubble up, and write documentation and changelogs.)

**Reply by t-zuehlsdorff** [reply]
How is it going @tarsius ? :)

**Reply by tarsius** [reply]
https://emacsair.me/2024/08/09/magit-4.0/
